src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -1715,6 +1715,7 @@ class CaseExpressionSegment(BaseSegment):
                             "FunctionSegment"
                         ),  # WHERE (a, substr(b,1,3)) IN (select c,d FROM...)
                         Ref("LiteralGrammar"),  # WHERE (a, 2) IN (SELECT b, c FROM ...)
+                        Ref("LocalAliasSegment"),   # WHERE (LOCAL.a, LOCAL.b) IN (…)
```

Comment:
Probably should keep eclipses consistent (damn auto correct!)

```suggestion
                        Ref("LocalAliasSegment"),   # WHERE (LOCAL.a, LOCAL.b) IN (…)
```